generic/tests/linux_stress.py

```diff
@@ -0,0 +1,203 @@
+import time
+import logging
+import aexpect
+
+from avocado.utils import download
+
+from virttest import data_dir
+from virttest import utils_misc
+from virttest import error_context
+
+
+class Stress(object):
+    """
+    Stress function for linux guest
+    """
+
+    def __init__(self, params, env):
+        """
+        Initialize StressTest object
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment
+        """
+        self.params = params
+        self.dst_path = "/home"
```

Comment:
I'd prefer to set the path in cfg.

Reply:
@huangyum If set it in cfg, use it as sub_test will require main test set the parameters in its cfg too, set it here could avoid redundant configuration, anyway it is not important where the path is.

Reply:
I'm not sure if it is hard coding. Maybe we could get if from params but with a default value.